src/extension/background-script/actions/accounts/add.ts

```diff
@@ -31,7 +17,10 @@ const add = async (
 
   const accountId = uuidv4();
   newAccount.config = encryptData(newAccount.config, password);
-  accounts[accountId] = newAccount;
+  accounts[accountId] = {
```

Comment:
Are we allowed to mutate state directly, or should we use a new accounts object?

Reply:
Just used it like it were. Updated now.